js/main.js

```diff
@@ -1,28 +1,118 @@
+$('document').ready(function(){
+
+
 /*
 Apple
 The apple tree (Malus pumila, commonly and erroneously called Malus domestica) is a deciduous tree in the rose family best known for its sweet, pomaceous fruit, the apple. It is cultivated worldwide as a fruit tree, and is the most widely grown species in the genus Malus. The tree originated in Central Asia, where its wild ancestor, Malus sieversii, is still found today. Apples have been grown for thousands of years in Asia and Europe, and were brought to North America by European colonists. Apples have religious and mythological significance in many cultures, including Norse, Greek and European Christian traditions.
+*/
+
 
+$('.apple').click(function() {
+ 		removeActive();
```

Comment:
It looks like you forgot to actually define this function anywhere. This is preventing the code from working because it doesn't know about a function called `removeActive`.  You'll need to define the function in order for this to work. e.g.

```js
function removeActive() {
  $('li').removeClass('active');
}
```